File: Relay/Arduino/FourPortRelay/CommandArgs.cpp
#include "CommandArgs.h"

using namespace Sannel::Relay::Command;

CommandArgs::CommandArgs()
{
}

CommandType CommandArgs::Get_Type()
{
	return this->type;
}

void CommandArgs::Set_Type(CommandType type)
{
	this->type = type;
}

Command CommandArgs::Get_Command()
{
	return this->command;
}

void CommandArgs::Set_Command(Command com)
{
	this->command = com;
}

byte* CommandArgs::Get_Value()
{
	return this->value;
}

void CommandArgs::Set_Value(byte* val)
{
	this->value = val;
}

File: Relay/Windows Phone/Sannel.Relay.Command/CommandArgs.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Sannel.Relay.Command
{
	public class CommandArgs
	{
		public CommandType Type
		{
			get;
			set;
		}

		public Command Command
		{
			get;
			set;
		}

		public byte[] Value
		{
			get;
			set;
		}
	}
}
